inspect_return_value.py

```diff
@@ -0,0 +1,170 @@
+
+  Please note: the various bash style boolean commands are listed as follows-
+  * bash-style-boolean-operator
+    --eq ... equal
+    --ne ... not equal
+    --lt ... less than
+    --le ... less than or equal
+    --gt ... greater than
+    --ge ... greater than or equal
+
+  General usage:
+  python inspect_return_value.py -l <path/to/the/link/file> -o <operator> <integer>
+
+"""
+import os
+import sys
+import argparse
+import in_toto.log
+from in_toto.models.link import Link as link_import
+import securesystemslib.exceptions
+
+
+def inspect_return_value(link, operator, integer):
+    """
+    <Purpose>
+      A function which performs the inspection as described above depending on various arguments.
+
+    <Arguments>
+      link:
+        the path to the link file
+
+      operator:
+        bash style boolean operator
+
+      integer:
+        the integer to which the return value should be compared
+        Note: the comparison happens treating the integer as a consequent,
+        for example: given "lt" as an operator, it will inspect whether
+        (the return value) <= (the given integer)
+
+    <Exceptions>
+      None
+
+    <Returns>
+      Boolean
+
+    """
+
+    imported_link = link_import.read_from_file(link)
+    if operator == 'eq':
+      if integer == imported_link.return_value:
+        return True
+
+    elif operator == 'ne':
+      if integer != imported_link.return_value:
+        return True
+
+    elif operator == 'lt':
+      if integer > imported_link.return_value:
+        return True
+
+    elif operator == 'le':
+      if integer >= imported_link.return_value:
+        return True
+
+    elif operator == 'gt':
+      if integer < imported_link.return_value:
+        return True
+
+    elif operator == 'ge':
+      if integer <= imported_link.return_value:
+        return True
+
+    return False
+
+def parse_args():
+
+    """
+    <Purpose>
+      A function which parses the user supplied arguments.
+
+    <Arguments>
+      None
+
+    <Exceptions>
+      None
+
+    <Returns>
+      Parsed arguments (args object)
+
+    """
+    parser = argparse.ArgumentParser(
+        description="Inspects the return value of a step")
+
+    lpad = (len(parser.prog) + 1) * " "
+
+    parser.usage = ("\n"
+                    "%(prog)s --link <path to link metadata>\n{0}"
+                    "[--<operator>]\n{0}"
+                    "<integer>\n\n"
+                    .format(lpad))
+
+    in_toto_args = parser.add_argument_group("in-toto-inspection options")
+
+    in_toto_args.add_argument("-l", "--link", type=str, required=True,
+                              help="Link metadata file to use for inspection "
+                              "of the step")
+
+    in_toto_args.add_argument("-o", "--operator", choices=['eq', 'ne',
+                              'lt', 'le', 'gt', 'ge'], type=str,
+                              required=True, help="The boolen operator "
```

Comment:
Actually, only AND, OR, NOT, and AND NOT are boolean operators (I know I called them so in the issue). Maybe we should rename them to logical operators.